_documentation/user/flags/ricochet.md

```diff
@@ -5,4 +5,12 @@ type: Good
 summary: Shots bounce off walls. Don't shoot yourself!
 ---
 
-...
+The Ricochet (R) flag allows the player to fire shots that can bounce off of the objects they hit. The direction of the bounce is determined by the angle under which the platform is hit. An exception are objects with the attributes `shootthrough` or `passable`, where the result is the bullet passing through the object.
+
+A lot of maps utilize the default ricochet option to already allow the effect of Ricochet without the tanks being required to hold the specific flag for it. The Ricochet flag holds meaning and effect on maps that do not utilize said option.
+
+## Tactics
+
+Ricochet is potent on maps filled with objects, usually boxes, behind which a tank can hide from direct enemy fire, all while attempting to shoot them down by using the objects as proxies off of which own bullets bounce in hopes of hitting the opponent.
```

Comment:
```suggestion
Ricochet is potent on maps filled with objects behind which a tank can hide from direct enemy fire, while attempting to shoot the opponent by bouncing shots off objects.
```

Reply:
@jwmelto can you elaborate as to why you removed the mention of boxes? They are notable for having straight walls, allowing for commonly 'useful' ricochet, contrary to most pyramids. Note that this is the `Tactics` section.

Otherwise good change suggestion.

Reply:
Ricochet is feasible and useful on maps with pyramids also. In fact, once can use ricochet off the angular side of a pyramid to hit a tank above you. I just didn't seen a need to limit the discussion to boxes.

Reply:
That is true; will change.